tools/test-static-code

```diff
@@ -80,4 +81,16 @@ else
     echo "ok 4 unsafe-security-policy"
 fi
 
+#
+# Bad paths or modifications in patternfly for fonts
+#
+
+if grep -R "\.\./fonts/OpenSans\|fonts/.*eot\|truetype" $distdir/dist/*/*.css >&2; then
+    echo "ERROR: invalid patternfly fonts paths found"
```

Comment:
This caused the endless loop, due to confusing TAP. This message needs to go to `>&2`.